### DISCUSSION TITLE
Bad Request: failed to process audio: corrupt or unsupported data // occur error

### DISCUSSION THREAD

**Discussion by Kang0228** [deepgram/community#533] [General help] [answered]
I am using Transcribe a Local File.
However, I get the error Bad Request: failed to process audio: corrupt or unsupported data. I've tried extensions mp3, mp4, wav, and raw, but I get the error. What is the problem? 

curl \
  --request POST \
  --header 'Authorization: Token YOUR_DEEPGRAM_API_KEY' \
  --header 'Content-Type: audio/wav' \
  --data-binary @youraudio.wav \
  --url 'https://api.deepgram.com/v1/listen'

request_id: '1df7db03-8de2-4b84-b874-8842c06c2e77'

**Resolved.**

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment]
I think you have another post in the discussions here, try and remove the content type specifying the media type.

**Comment by davidvonthenen** [comment, marked as answer]
Other post:
https://github.com/orgs/deepgram/discussions/531
